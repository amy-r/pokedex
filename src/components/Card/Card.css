.Card {
  background-color: #FEFAF1;
  width: 100px;
  margin: 10px;
}