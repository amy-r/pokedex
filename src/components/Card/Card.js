import React, { Component } from 'react';
import PropTypes, { shape, func, string } from 'prop-types';
import { connect } from 'react-redux';
import './Card.css'

class Card extends Component {
  constructor(){
  super()
  }
  render() {
    return (
      <div className= 'Card'>
      <h3> {this.props.name} </h3>
    </div>
    )
  }
}

export default Card;