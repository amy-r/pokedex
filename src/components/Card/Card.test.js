import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer'

describe('Card', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('matches a snapshot', () => {

    const card = renderer.create(<Card />).toJSON()
    expect(card).toMatchSnapshot();  
  });
})

  