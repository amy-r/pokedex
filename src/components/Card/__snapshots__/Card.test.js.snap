// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card matches a snapshot 1`] = `
<div
  className="Card"
>
  <h3>
     
     
  </h3>
</div>
`;
